### PR TITLE
fix analytics

### PR DESCRIPTION
**Pull request by stefanpenner**
cc @twokul

### CODE REVIEW

**Comment by rwjblue** [comment]
:+1:

File: lib/cli/index.js [hunk 1]
```diff
@@ -40,7 +40,9 @@ function cli(options) {
   var config = new Yam('ember-cli');
   var leekOptions;
 
-  var disableAnalytics = options.cliArgs && options.cliArgs.indexOf('--disable-analytics') || config.get('disableAnalytics');
+  var disableAnalytics = options.cliArgs &&
+    (options.cliArgs.indexOf('--disable-analytics') > -1 )||
```
Comment (jakecraige) [inline lib/cli/index.js]: missing a space here left of `||`

Comment (rwjblue) [inline lib/cli/index.js]: I added a commit fixing the spacing.

Comment (stefanpenner) [inline lib/cli/index.js]: heh, i also did :P

**Comment by twokul** [comment]
@stefanpenner thanks man

**Pull request merged**